src/utils.jl

```diff
@@ -0,0 +1,49 @@
+StatsBase.describe(X::NullableVector) = StatsBase.describe(STDOUT, X)
+
+function StatsBase.describe(io::IO, X::NullableVector{<:Real})
+    if allnull(X)
+        println(io, " * All null * ") && return
```

Comment:
I don't particularly mind either way, though I guess the second option there without the explicit "all null" part would be preferable for consistency.